src/onion_messenger.rs

```diff
@@ -78,11 +78,11 @@ impl Logger for MessengerUtilities {
         let args_str = record.args.to_string();
         match record.level {
             Level::Gossip => {}
-            Level::Trace => trace!("{}", args_str),
-            Level::Debug => debug!("{}", args_str),
-            Level::Info => info!("{}", args_str),
-            Level::Warn => warn!("{}", args_str),
-            Level::Error => error!("{}", args_str),
+            Level::Trace => trace!(target: "ldk", "{}", args_str),
```

Comment:
Just a nit, but since we use the "lndk" string so many times, I think it'd be worth making that a const that we reference here